src/administration/integrations/github.md

```diff
@@ -22,8 +22,7 @@ profile. Simply go to your account page on GitHub and click
 Select the *Personal access tokens* tab and click on [Generate new
 token](https://github.com/settings/tokens/new).
 
-Give it a description and then ensure the token has the following
-scopes:
+Give it a description and then ensure the token has the scopes as follows:
```

Comment:
This change seems backward.  "Has the scopes" doesn't mean much in context.  "Has the following scopes" is more descriptive of the process.